plugin.video.telebaern/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.telebaern" name="TeleBärn" version="0.9.0" provider-name="Alexander Seiler">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.azmedien" version="0.9.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">TeleBärn media library</summary>
+    <summary lang="de_DE">TeleBärn Mediathek</summary>
+    <description lang="en_GB">Watch all the shows of the local Swiss TV station TeleBärn.</description>
+    <description lang="de_DE">Sehen Sie alle Sendungen des lokalen schweizer Fernsehrsender TeleBärn.</description>
+    <language>de</language>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
+    <website>https://www.telebaern.tv</website>
+    <email>seileralex@gmail.com</email>
+    <source>https://github.com/goggle/plugin.video.telebaern</source>
+    <news>v0.9.0 (2018-10-26)
+ - First release.</news>
+    <assets>
+        <icon>resources/icon.png</icon>
```

Comment:
It is advised not to use TM icons and images if addons are not officially developed by the respective provider. However this is not a hard rule, just a recomendation.